webpack.prod.babel.js

```diff
@@ -86,7 +84,8 @@ const commonConfig = merge([
   parts.loadJS({
     include: paths.src,
     exclude: /node_modules/
-  })
+  }),
+  parts.copyImages()
 ]);
 
-module.exports = commonConfig;
+module.exports = commonConfig;
```

Comment:
__[ESLint]__ Error - Newline required at end of file but not found. (eol-last) ([link](https://sider.review/gh/repos/163017290/pulls/16#issue-4573293))

<sub>posted by Sider</sub>